src/commands/games/blackjack.ts

```diff
@@ -190,13 +182,17 @@ class BlackjackCommand extends BaseCommand {
     adjustCoinBalanceByUserId(playerId, balanceChange, UserCoinEvent.Blackjack);
   }
 
-  async onRun(message: CommandoMessage, args: { bet: number }): Promise<Message> {
-    const { bet } = args;
+  async messageRun(message: Message, args: Args): Promise<Message> {
+    const bet = args.finished
+      ? DEFAULT_BET
+      : await args.rest('integer').catch(() => 'please enter a valid bet amount.');
```

Comment:
in general, plz add one-line comment in sorta larger functions plz
makes my life reviewing a bit easier :P